fudge-ratio/src/main/webapp/code/mainScreen.js

```diff
@@ -0,0 +1,90 @@
+const mainText = `
+  <div id="timer-contents">
+    <h3 id="title"></h3>
+    <p id="timer-display">00:00:00</p>
+    <div class="timer-buttons">
+      <div>
+        <button id="start-stop">Start</button>
+        <button id="finish" disabled>Finish</button>
+      </div>
+      <div>
+        <button id="clear">Reset all</button>
+      </div>
+      <p id="result"></p>
+      <p id="average"></p>
+    </div>
+  </div>
+  `;
+
+import { get, parent, createFromText } from "./utils.js";
+import { KEYS } from "./save.js";
+import Timer from "./time.js";
+
+const showMain = () =>
+  new Promise((resolve, _) => {
+    const timerStorage = localStorage.getItem(KEYS.TIME_KEY);
+    const timer = new Timer();
+
+    parent.appendChild(createFromText(mainText));
+
+    const activity = localStorage.getItem(KEYS.ACTIVITY_KEY);
+    get("title").textContent = "You are doing : " + activity;
+
+    const text = get("timer-display");
+    const startStop = get("start-stop");
+    const finish = get("finish");
+    let id = null;
+    startStop.onclick = () => {
+      if (startStop.textContent !== "Stop") {
+        id = setInterval(() => (text.textContent = timer.addSeconds(1).getText()), 1000);
+        startStop.textContent = "Stop";
+        startStop.style.backgroundColor = "var(--red-color)";
+        text.style.fontWeight = 700;
+        finish.disabled = false;
+        if (!timerStorage) localStorage.setItem(KEYS.TIME_KEY, Date.now());
+      } else {
+        startStop.textContent = "Continue";
+        text.style.fontWeight = 400;
+        startStop.style.backgroundColor = "var(--green-color)";
+        clearInterval(id);
+        localStorage.removeItem(KEYS.TIME_KEY);
+      }
+    };
+
+    if (timerStorage) {
+      timer.seconds = Math.round((Date.now() - timerStorage) / 1000);
+      startStop.click();
+    }
+
+    get("clear").onclick = () => {
+      localStorage.clear();
+      location.reload();
+    };
+
+    get("finish").onclick = async () => {
+      startStop.click();
+      const start = new Timer().getText();
+      const end = timer.getText();
+      const expected = new Timer(localStorage.getItem(KEYS.EXPECTED_KEY)).seconds;
+
+      const response = await fetch("/fudgeRatio", {
+        method: "POST",
+        headers: { Accept: "application/json", "Content-Type": "application/json;charset=UTF-8" },
+        body: JSON.stringify({ start, end, expected }),
+      });
+
+      const { ratio } = await response.json();
+      get("result").textContent = `Next time think about allocating ${ratio.toFixed(2)}x 
+      ${ratio > 1 ? "more" : "less"}
+      for ${activity}`;
```

Comment:
This is me being very picky, but the reason I had a method before called format task was just so that when we display  "Next time think about allocating ... "  the sentence is is just a little nicer (theres picture of that old function below as a reminder). For example, if the person selects "English"  as their task ( under the homework category) then the final sentence would say "think about allocating .. for english **homework** . Another small thing is that I think it would be more effective to actually display the recommended **time** in the sentence because for a lot of people, its hard to conceptualize the ratio, especially if it is a decimal. In your example gif, I think some people would have a hard time with really visualizing  .62 of their original time (and remembering this in the future to help them change their behavior) without going off on their own and using a calculator to convert it to actual minutes/hours. Let me know what you think about this!
<img width="557" alt="Screen Shot 2020-04-16 at 10 58 02 AM" src="https://user-images.githubusercontent.com/49773006/79472203-89804a00-7fd1-11ea-84a0-ff6c0d402672.png">

Reply:
This is a really GREAT comment, and I completely agree with you.
https://gph.is/2aUc9VT

The general idea is that this need improvement, and I am completely  on your side, to be totally honest, I refactored the excepted time, the activity selector and then the main screen, so I was to tired to create a new "screen" for the final result, but I think the final result deserve one, so my idea was that when your finish the time, a transition occurs where you have a big message with the result, maybe some analytics but the most important part was that message "PLEASE ALLOCATE ..." and then maybe a couple of buttons to share the app or start a new timer.

In part I left that undo because I was to tired but also because I remembered that @gilemos, wanted to to do some frontend, so I thought that this was perfect for a task this week, create a final screen with the correct message (as you said in a more readable way, or Human way, maybe is time to get back to life the ``getHuman`` 😅), I think i left a comment in my last PR mentioning this exact situation.

So TLDR; I agree with the 2 main points, we should use your idea to create a better message and give that message in a more readable way.

I thought that @gilemos could do that, but if she is too busy maybe you or me could do it, I see no problem doing it myself :3

<img width="914" alt="Screen Shot 2020-04-16 at 4 53 35 PM" src="https://user-images.githubusercontent.com/23039467/79510338-d2042b80-8002-11ea-9071-68975d43ff65.png">

Reply:
Thanks @yaaowusu00 for taking the time for really checking the changes and giving your honest opinions

Reply:
I am really busy with final exams so I'm not sure if I can get to it 😞